filament/backend/src/metal/MetalDriver.mm

```diff
@@ -754,8 +754,15 @@
     return false;
 }
 
-bool MetalDriver::isStereoSupported() {
-    return true;
+bool MetalDriver::isStereoSupported(backend::StereoscopicType stereoscopicType) {
+    switch (stereoscopicType) {
+    case backend::StereoscopicType::INSTANCED:
+        return true;
+    case backend::StereoscopicType::MULTIVIEW:
+        // TODO: implement multiview feature in Metal.
+    default:
```

Comment:
personally I prefer not to have a `default` because if we ever add an enum type, the compiler will be able to warn us that not all cases are handled. This saved me before :-)

Reply:
Ah that's actually better. Let me remove it.